client/src/resources/style-constants.js

```diff
@@ -20,6 +20,20 @@ export const link = "#2196f3"; // color for links
 export const highlightedLink = "#35baf6"; // color of links on hover
 export const errorRed = "#FF0002";
 
+/*
+Swatches (shades of colors).
+*/
+export const primarySwatch = {
```

Comment:
We can switch to use these colors swatches if you want to define them all. I've been using the theme1 as the main color, and then theme2 is a slighter darker shade for the button hover colors. I'm fine either way, but I do want to be consistent throughout the entire project.

Reply:
Good point. I left a comment in the main thread about this. LMK